### PR TITLE
[FIX] look like customer field has been named "client" in the pos client

### PR DESCRIPTION
**Pull request by houssine78**
this PR fixes the customer field name

### CODE REVIEW

**Comment by houssine78** [comment]
@robinkeunen @remytms old PR here. I don't remember for which customer is related but it would be great if you could have a look

File: pos_custom_receipt/static/src/xml/pos.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
         <t t-jquery="t[t-if='receipt.company.email']" t-operation='replace'/>
         <t t-jquery="t[t-if='receipt.cashier']" t-operation='after'>
         	 <t t-if='receipt.partner'>
```
Comment (robinkeunen) [inline pos_custom_receipt/static/src/xml/pos.xml]: You probably need to dix this line too

**Review by robinkeunen — CHANGES_REQUESTED** [review]
I think you need to fix the previous if-statement :-)

File: pos_custom_receipt/static/src/xml/pos.xml (hunk 1 above)
Comment (houssine78) [reply pos_custom_receipt/static/src/xml/pos.xml]: done !

**Review by robinkeunen — APPROVED** [review]

**Pull request merged**